LICENSE.md

```diff
@@ -1,33 +1,49 @@
+> of this software and associated documentation files (the "Software"), to deal
+> in the Software without restriction, including without limitation the rights
+> to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+> copies of the Software, and to permit persons to whom the Software is
+> furnished to do so, subject to the following conditions:
+>
+> The above copyright notice and this permission notice shall be included in all
+> copies or substantial portions of the Software.
+>
+> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+> IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+> FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+> AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+> LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+> OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+> SOFTWARE.
 
-  * Volker Diels-Grabsch
-  * Mark Brand
-  * Tony Theodore
-  * Martin Gerhardy
-  * Tiancheng "Timothy" Gu
-  * Boris Nagaev
-  * ... and many other contributors
+MXE distributes code from the following projects in the `ext/` directory:
 
-(contact via the project mailing list)
+- [GNU config.guess][config.guess] [GPL3+ with Autoconf exception]
+- [GNU Make Standard Library][gmsl] [BSD 3-clause]
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+The [packages][packages] downloaded by MXE (ending up in `pkg/`) each
+have their own licence(s). Please see the individual source trees and/or
+websites for further details.
 
+[contributors]:https://github.com/mxe/mxe/graphs/contributors
+[mailing-list]:https://lists.nongnu.org/mailman/listinfo/mingw-cross-env-list
+[issue-tracker]:https://github.com/mxe/mxe/issues
+[config.guess]:http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD
+[gmsl]:https://sourceforge.net/p/gmsl/gmsl/ci/master/tree/__gmsl
```

Comment:
Could you provide links to revisions of config.guess and gmsl in use by MXE, please?